packages/orbit-components/src/BadgeList/index.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+
+import type { Props } from "./index";
+
+const StyledBadgeList = styled.ul`
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+`;
+
+const BadgeList = ({ children, dataTest }: Props): React.Node => {
+  return <StyledBadgeList data-test={dataTest}>{children}</StyledBadgeList>;
```

Comment:
wdy about applying `role="presentation"` on the list?

Reply:
Doesn't using `role="presentation"` hide the element and children of it from readers?

Reply:
i think it should affect `ul` and `li`, but not the content of them. `[role="presentation"] negates the implicit 'list'
and 'listitem' role semantics but does not affect the contents.`. But i just asked, I'm not sure about it